### PR TITLE
Added usage example for tf.math.sigmoid

### PR DESCRIPTION
**Pull request by BashirSbaiti**
Added a brief example showing how to use tf.math.sigmoid

### CODE REVIEW

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -3122,6 +3122,13 @@ def sigmoid(x, name=None):
 
   Returns:
     A Tensor with the same type as `x`.
+    
+  Usage Example:
+    '''python
```
Comment (kyscg) [inline tensorflow/python/ops/math_ops.py]:
```suggestion
    ```python
```

File: tensorflow/python/ops/math_ops.py [hunk 2]
```diff
@@ -3122,6 +3122,13 @@ def sigmoid(x, name=None):
...
+    import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/math_ops.py]:
```suggestion
    
```

Comment (kyscg) [reply tensorflow/python/ops/math_ops.py]: No need to import tensorflow

File: tensorflow/python/ops/math_ops.py [hunk 3]
```diff
@@ -3122,6 +3122,13 @@ def sigmoid(x, name=None):
...
+    x = tf.constant([125.92, -76.32, 44.12, -87.3, -82.72, 0.0, 112.23], dtype=tf.float32)
+    x = tf.sigmoid(x) # x = [1.0, 0.0, 1.0, 0.0, 0.0, 0.5, 1.0,]
+    '''
```
Comment (kyscg) [inline tensorflow/python/ops/math_ops.py]:
```suggestion
    ```
```

Comment (kyscg) [reply tensorflow/python/ops/math_ops.py]: Backticks, not quotes.

**Review by kyscg — CHANGES_REQUESTED** [review]

**Comment by rahul2240** [comment]
Hi @BashirSbaiti, thanks for the PR. I have noticed that we have 4 commits for a single example that you have added. We should do it in a single commit so that it becomes easier to understand what that commit does. You can squash the extra commits, feel free to ask if you face any problem.

**Comment by BashirSbaiti** [comment]
(Sorry if this is a dumb question because I am new to github)
Does this mean I should open a new PR or edit this one somehow?

**Comment by rahul2240** [comment]
@BashirSbaiti It's a good question :smile:, You don't have to open a new PR. We will work on this together but before that, I would suggest going through [Squash Commit ](https://www.internalpointers.com/post/squash-commits-into-one-git)and then try it with your branch.

Here are some hints for you:
1. git rebase -i HEAD~4
2. For last 3 commit's change "pick" to "squash"
3. save changes and close editor
4. force push

**Comment by kyscg** [comment]
You don't have to open a new one. 

Run `git rebase -i <your-branch>`
You'll see all your commits starting with `pick`, keep the first one unchanged and change the rest to `squash`

**Comment by kyscg** [comment]
@googlebot I consent.

**Comment by BashirSbaiti** [comment]
@rahul2240 I did a rebase to drop the extra commits and just keep the one that is the combination of the original four but it isn't showing up here after I ran git push origin master. Do you know why?

**Comment by BashirSbaiti** [comment]
@googlebot I consent.

**Comment by rahul2240** [comment]
@BashirSbaiti I can see that you have worked on patch-2 branch, so you have to do the rebasing in that branch. Why did you use "git push origin master"? "git push origin patch-2" should be the command that you should use.

Here's the thing that you have to do now -
1. Make sure you are in patch-2 branch.
2. Check what's your last commit.
3. If it includes these 3 extra commits then do rebasing as suggested.
4. Otherwise, make the right changes in the file and use "git commit --amend" which will add the changes in your last commit.
5. git push origin patch-2 -f

**Comment by rthadur** [comment]
Closing this PR as this not against `master`, please open a new PR against `master`

**Pull request closed without merging**